### DISCUSSION TITLE
Save changes when using the Main Window without being asked for a name

### DISCUSSION THREAD

**Discussion by vitorvas** [ImageMagick/ImageMagick#4529] [Help]
Hello all,

I've been googling for some time and can't find an answer. I am a long time user of Image Magick using the command-line. 
However, now I need to work on a batch of images which need or not need to be rotated, right or left. This must be done manually.
Since I can use / to rotate the images, `Ctrl-D` to delete some, I just need to rotate and save with the same name. The shortcut `Ctrl-S` opens the dialog windows and I have to type a name. Unthinkable when processing a series of images to type the check and type the same name over and over again.

The `Help->Overview->Keyboard Accelerators` gives me a hint: `Cmd-A` does the trick of making changes permanent. Perfect! The only problem is that I can't find a tip on what is this "`Cmd`¨. Not the control key, nor the alt key, nor shift, nor a combination. So, anybody can help me with that?

Thank you very much,

Vitor

**Comment by urban-warrior** [comment]
Thanks for the problem report.  We can reproduce it and will have a patch to fix it in the GIT main branch @ https://github.com/ImageMagick/ImageMagick later today.  The patch will be available in the beta releases of ImageMagick @ https://imagemagick.org/download/beta/ by sometime tomorrow.
